autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/AutoscaleClusterState.java

```diff
@@ -9,25 +10,48 @@
 public class AutoscaleClusterState {
 
     @ApiModelProperty(ClusterAutoscaleState.ENABLE_AUTOSCALING)
-    private boolean enableAutoscaling;
+    private Boolean enableAutoscaling;
```

Comment:
This is an optional parameter, and a null value indicates that the parameter is not being changed. Instead a different field in the class may be changed.

Reply:
These look like independent configuration values, why not have a separate endpoint for controlling it?
It will even be more flexible if you need to pass some form of configuration. Then the `Boolean` won't be needed.

Reply:
Having a separate endpoint for this would result in 2 separate API calls being made from the CDPCLI, hence it was decided to bring both these values together and toggle them independently using `Boolean`. Discussion on [CB-15140](jira.cloudera.com/browse/CB-15140) for reference: https://github.infra.cloudera.com/thunderhead/thunderhead/pull/6894#discussion_r122533

Reply:
The 2 API calls from CDP CLI risk the operation failing midway, leaving the cluster in an inconsistent state.